### ISSUE TITLE
docs: 12 committed renders publish the pre-#2090 vacuous DeadFitWarning over healthy fits

### ISSUE THREAD

**Issue by suchethac** [suchethac/tengri#2113]
## Observation

Twelve committed notebook renders publish the text `DeadFitWarning: dead fit: parameter(s) ...` over fits that are healthy. Example — `docs/spine/00_quickstart.ipynb` cell 11 (fit converged, `R̂ = 1.0005`):

```
src/tengri/inference/backends/mcmc/hmc.py:325: DeadFitWarning: dead fit: parameter(s) 'dust_Rv', 'dust_bump_strength', 'dust_delta', 'dust_f_obscuration', 'dust_slope', 'met_alpha_fe', 'met_logzsol_scatter', 'noise_dof', 'noise_frac_cal', 'redshift', 'sfh_dpl_age_gyr', 'sigma_v_kms' have 1 unique draw in 300 samples. R-hat cannot detect this. Remedies: ...
```

Every parameter it names is pinned. This is the vacuous pre-#2090 form of the dead-fit warning — it fired on Fixed parameters, which trivially have one unique draw — and #2090 (`3bc0506b1`, 2026-08-30) fixed the code to skip Fixed parameters. The renders were executed 2026-08-21 (#2034) and have not been re-executed since, so the published pages still tell readers their converged example fits are dead.

## Affected files (grep `DeadFitWarning` in committed .ipynb outputs, origin/main)

- `notebooks/00_quickstart.ipynb` + `docs/spine/00_quickstart.ipynb` (cell 11)
- `notebooks/01_why_jax.ipynb` + `docs/spine/01_why_jax.ipynb` (cell 9)
- `notebooks/06_fitting_spectroscopy.ipynb` + `docs/spine/06_fitting_spectroscopy.ipynb` (cell 9)
- `notebooks/07_joint_photo_spec.ipynb` + `docs/spine/07_joint_photo_spec.ipynb` (cell 9)
- `notebooks/11_catalog_fits.ipynb` + `docs/spine/11_catalog_fits.ipynb` (cell 9)
- `docs/spine/experimental/jwst_nonparametric_fits.ipynb` (cells 7 and 9)
- `docs/spine/experimental/stochastic_sfh_recovery.ipynb` (cell 19)

(`05_fitting_photometry`'s twins also carry the warning, but that render is a genuinely dead fit — tracked separately as #2095.)

## Ask

Re-execute these renders on a main that contains #2090 (and, once merged, #2110 — so a genuinely dead seed raises instead of rendering) and jupytext-sync in the same PR, per the two-ways-stale rules. Until then, the render-diagnostics guard (#2095's PR) carries these files as KNOWN-BAD entries citing this issue; re-rendering removes the warning text and the entries together. Mind the notebook OOM rules (one NUTS fit per process; `dense_mass_matrix=False` on D≥8) when re-executing.

**Comment by suchethac** [comment]
Claimed by orchestrator session tengri [c8861d], 2026-09-04 (post-#2128: #2090+#2110+#2132 are all on main, so a genuinely dead seed now raises during re-execution rather than rendering). Plan per the issue's ask: re-execute the 12 renders on main, jupytext-sync in the same PR, trim the render-diagnostics KNOWN-BAD entries for whatever re-renders cleanly, and report any notebook whose fit now refuses against its own tracking issue (e.g. #1999 for 06) instead of forcing it. Peers: PR #2130 confirmed disjoint (no spine/notebooks files).
